Dockerfile

```diff
@@ -31,93 +10,31 @@ COPY doc/datasets/popularize_churn.sql \
      /docker-entrypoint-initdb.d/
 VOLUME /var/lib/mysql
 
-# Install protobuf and protoc
-COPY scripts/docker/install-protobuf.bash /
-RUN /install-protobuf.bash
-
-# Need Java SDK to build remote parsers
-ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
-COPY scripts/docker/install-java.bash /
-RUN /install-java.bash
-# Make mvn compile quiet
-ENV MAVEN_OPTS -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-
-# Using the stable version of Hadoop
-ENV HADOOP_VERSION 3.2.1
-ENV PATH /opt/hadoop-${HADOOP_VERSION}/bin:/usr/local/go/bin:/go/bin:$PATH
-COPY scripts/docker/install-hadoop.bash /
-RUN /install-hadoop.bash
-
-# Python 3, TensorFlow 2.0.0, etc
-COPY scripts/docker/install-python.bash /
-RUN /install-python.bash
-
-# Go, goyacc, protoc-gen-go, and other Go tools
-ENV GOPATH /root/go
-ENV PATH /usr/local/go/bin:$GOPATH/bin:$PATH
-COPY scripts/docker/install-go.bash /
-RUN /install-go.bash
-
-# ODPS
-COPY scripts/docker/install-odps.bash /
-RUN /install-odps.bash
-
-# ElasticDL and kubectl
-COPY scripts/docker/install-elasticdl.bash /
-RUN /install-elasticdl.bash
+# Install the Python source code.
+# TODO(yi): It seems that we don't need to build python/couler into a wheel.
+COPY python /usr/local/sqlflow/python
+ENV PYTHONPATH=/usr/local/sqlflow/python:$PYTHONPATH
 
-# The SQLFlow magic command for Jupyter.
-ENV IPYTHON_STARTUP /root/.ipython/profile_default/startup/
-COPY scripts/docker/install-jupyter.bash /
-COPY scripts/docker/js /js
-RUN /install-jupyter.bash
+# Install Couler, Fluid, and model zoo.
+COPY build/*.whl /usr/local/sqlflow/python/
+RUN pip install /usr/local/sqlflow/python/*.whl
```

Comment:
`RUN pip install /usr/local/sqlflow/python/*.whl && rm -f /usr/local/sqlflow/python/*.whl` to make the image smaller